src/sidebar.js

```diff
@@ -36,34 +36,41 @@ module.exports = {
       type: "category",
       label: "Identities and Sessions",
       items: [
-        "kratos/ory-kratos-intro",
+        "identities/native-browser",
+        "security-model",
         {
           type: "category",
-          label: "Managing identities",
+          label: "Sign in & registration",
```

Comment:
"Sign in" is a verb. You'd want to use "Sign-in", the noun here. 
However, the Microsoft Manual of Style suggests avoiding using "sign-in": https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/sign-in-sign-out

Therefore my suggestion would be to use "Login" as an unambiguous, widely used term.

Reply:
SGTM

Reply:
```suggestion
          label: "Sign-in & registration",
```